src/sentry/api/bases/sentryapps.py

```diff
@@ -118,12 +126,24 @@ class SentryAppPermission(SentryPermission):
     }
 
     published_scope_map = {
-        'GET': (),  # Public endpoint.
+        # Public endpoint.
+        'GET': ('event:read',
+                'event:write',
+                'event:admin',
+                'project:releases',
+                'project:read',
+                'org:read',
+                'member:read',
+                'team:read',),
```

Comment:
This isn't going to make it public, though, because if an anonymous user is making the request, they will not be authenticated, meaning there won't be a token/session with any scopes associated with it.